app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only backs up APK files from the source directory. (Select <b>Source</b> first.)</string>
+    <string name="backup_data_description">Back up internal data folders (for inclusion in backups) by default.</string>
```

Comment:
The sentence has become confusing even to me. There's no reason to add the phrase inside the brackets. Also it should be “Back**s** up internal data...”. As I said in the past, “Enable backup for data folders. When enabled, internal data folders are backed up by default” is the best explanation for this flag. Every alternatives you proposed thus far don't convey the original meaning.